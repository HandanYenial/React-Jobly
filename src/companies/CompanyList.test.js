import React from "react";
import { render } from "@testing-library/react";
import Companies from "./CompanyList";

it ("renders without crashing" , function(){
    render(<Companies/>);
});

it("matches with snapshot", function(){
    const { asFragment } = render(<Companies/>);
    expect(asFragment()).toMatchSnapshot();
});