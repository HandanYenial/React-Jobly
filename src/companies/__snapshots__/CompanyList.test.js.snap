// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches with snapshot 1`] = `
<DocumentFragment>
  <div
    class="LoadingSpinner"
  >
    Loading...
  </div>
</DocumentFragment>
`;
